test/grouping.jl

```diff
@@ -717,37 +735,33 @@ end
             @test f(col => identity, gd) == f(d -> (c_identity=d.c,), gd)
             @test f(col => x -> (z=x,), gd) == f(d -> (z=d.c,), gd)
 
-            @test f((xyz = col => sum,), gd) ==
+            @test f(col => sum => :xyz, gd) ==
                 f(d -> (xyz=sum(d.c),), gd)
-            @test f((xyz = col => x -> sum(x),), gd) ==
+            @test f(col => (x -> sum(x)) => :xyz, gd) ==
                 f(d -> (xyz=sum(d.c),), gd)
-            @test f((xyz = col => x -> (sum(x),),), gd) ==
+            @test f(col => (x -> (sum(x),)) => :xyz, gd) ==
                 f(d -> (xyz=(sum(d.c),),), gd)
-            @test_throws ArgumentError f((xyz = col => x -> (z=sum(x),),), gd)
-            @test_throws ArgumentError f((xyz = col => x -> DataFrame(z=sum(x),),), gd)
-            @test_throws ArgumentError f((xyz = col => x -> (z=x,),), gd)
+            @test_throws ArgumentError f(col => (x -> (z=sum(x),),) => :xyz, gd)
+            @test_throws ArgumentError f(col => (x -> DataFrame(z=sum(x),),) => :xyz, gd)
+            @test_throws ArgumentError f(col => (x -> (z=x,),) => :xyz, gd)
             @test_throws ArgumentError f(col => x -> (z=1, xzz=[1]), gd)
 
             for wrap in (vcat, tuple)
-                @test f(wrap(col => sum), gd) ==
-                    f(d -> (c_sum=sum(d.c),), gd)
-                @test f(wrap(col => x -> sum(x)), gd) ==
-                    f(d -> (c_function=sum(d.c),), gd)
-                @test f(wrap(col => x -> (sum(x),)), gd) ==
-                    f(d -> (c_function=(sum(d.c),),), gd)
-                @test_throws ArgumentError f(wrap(col => x -> (z=sum(x),)), gd)
-                @test_throws ArgumentError f(wrap(col => x -> DataFrame(z=sum(x),)), gd)
-                @test_throws ArgumentError f(wrap(col => x -> (z=x,)), gd)
-                @test_throws ArgumentError f(wrap(col => x -> (z=1, xzz=[1])), gd)
+                @test_throws MethodError f(wrap(col => sum), gd)
```

Comment:
Maybe just drop that block. We can't test all functions that may exist. ;-)

Reply:
ok - removed; it will save some test-running time